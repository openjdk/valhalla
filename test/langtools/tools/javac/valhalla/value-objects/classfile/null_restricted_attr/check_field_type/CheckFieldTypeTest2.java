/* @test /nodynamiccopyright/
 * @summary [lw5] check that there can only be one NullRestricted attribute
 * @modules jdk.compiler/com.sun.tools.javac.api
 *          jdk.compiler/com.sun.tools.javac.main
 *          jdk.compiler/com.sun.tools.javac.util
 * @library /tools/lib
 * @compile --enable-preview -source ${jdk.version} NullRestrictedOnArray.jcod
 * @compile/fail/ref=CheckFieldTypeTest2.out --enable-preview -source ${jdk.version} -XDrawDiagnostics CheckFieldTypeTest2.java
 */

public class CheckFieldTypeTest2 {
    void m() {
        NullRestrictedOnArray v = new NullRestrictedOnArray();
    }
}
