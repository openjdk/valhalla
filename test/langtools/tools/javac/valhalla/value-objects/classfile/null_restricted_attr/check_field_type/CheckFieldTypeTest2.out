CheckFieldTypeTest2.java:13:9: compiler.err.cant.access: NullRestrictedOnArray, (compiler.misc.bad.class.file.header: NullRestrictedOnArray.class, (compiler.misc.attribute.not.applicable.to.field.type: NullRestricted, java.lang.String[]))
1 error
